Formula/opam.rb

```diff
@@ -1,104 +1,31 @@
 class Opam < Formula
-  desc "Package manager for OCaml"
+  desc "The OCaml package manager"
   homepage "https://opam.ocaml.org"
-  url "https://github.com/ocaml/opam/archive/1.2.2.tar.gz"
-  sha256 "3e4a05df6ff8deecba019d885ebe902eb933acb6e2fc7784ffee1ee14871e36a"
-  revision 4
+  url "https://github.com/ocaml/opam/releases/download/2.0.0/opam-full-2.0.0.tar.gz"
+  sha256 "9dad4fcb4f53878c9daa6285d8456ccc671e21bfa71544d1f926fb8a63bfed25"
   head "https://github.com/ocaml/opam.git"
 
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "07f2777f0dda170b36e409a6db773a5aae0e202e812127b388a05afaa89b3949" => :mojave
-    sha256 "b5e2621c1bca5f8374ee07ef878e5572e04debf9ba1d3aa4a2e16b8e26728e68" => :high_sierra
-    sha256 "cd52d891272efc754a838e8a08a4a7c5030ff908430c3ed1303a549cd1a4f73d" => :sierra
-    sha256 "74f8341302bb5a933276cff7f9dff7240ad59a4d968050674b63869d9963de7e" => :el_capitan
-  end
-
+  depends_on "glpk" => :build
   depends_on "ocaml" => :recommended
```

Comment:
It seems the `--without-ocaml` option is never used, so please remove `=> :recommended` and make it mandatory.

Reply:
I don't understand what you mean here. `--without-ocaml` option is handled in the `install` section. You can let OPAM also build OCaml, so it only is a build dependency if you don't want it to build. For running the `ocaml` package is not necessary.

I've adjusted that line to be more sensible, hope this is what you are envisioning.

Reply:
I decided to uncomplicate the whole thing - now it always depends on a system compiler to be built. This also prevents it from attempting to download a tarball (which oddly enough is already inside the `-full` tarball anyway).